### ISSUE TITLE
@return array not detected as UselessDocComment

### ISSUE THREAD

**Issue by jeroennoten** [slevomat/coding-standard#215]
I would expect that this phpDoc is detected as useless by the `TypeHintDeclaration. UselessDocComment` rule, however it is not reported.
```php
    /**
     * @return array
     */
     function foo(): array
     {
        // ...
     }
```
Note that I have disabled the `MissingTraversableReturnTypeHintSpecification`.

**Comment by kukulich** [comment]
I would have to check if the rule is disabled. I'm not sure if it's even possible. And it would also mean that I would have to read PHPCS configuration that can probably change in newer versions because it's internal.

So I will not try to fix this.
